### ISSUE TITLE
PytestUnknownMarkWarning: Unknown pytest.mark.datafiles with pytest 6.2.4

### ISSUE THREAD

**Issue by KirkDCO** [omarkohl/pytest-datafiles#18]
I see a similar issue with an earlier version of pytest (4.3.0) which appears to have been fixed with pytest v4.5.0.  But I'm using pytest v6.2.4 - is there something I'm missing?

**Comment by Arpafaucon** [comment]
Hi, same here 

```
============================================================================================================ test session starts =============================================================================================================
platform linux -- Python 3.8.10, pytest-6.2.4, py-1.10.0, pluggy-0.13.1
plugins: cov-2.12.1, dash-1.18.1, datafiles-2.0, anyio-3.2.1, mock-3.6.1
```

**Comment by simplogic86** [comment]
Have you tried registering the marker in your config file? Check out hf-krechan's Feb. 10 answer on the 4.3.0 issue thread. Worked for me.

**Comment by travipross** [comment]
![image](https://user-images.githubusercontent.com/17794229/153650066-1853a10f-6008-46e6-92dc-70ca3917b763.png)
:thinking:

**Comment by omarkohl** [comment]
Right you are @travipross and sorry about that! Too many things and I haven't needed the plugin myself in years, so yeah... I want to have a look at all the pull requests and issues in the next weeks. To be honest I can't believe that so many people appear to be using the plugin. I will try to do better!

**Comment by travipross** [comment]
@omarkohl haha don't take my comment too seriously at all. I was just being cheeky. Thank you for contributing to the open source world! This is a solid plugin.

**Comment by omarkohl** [comment]
Should be fixed in commit f95315ee2dff86ffeb17a41c203d54c1fc1405f0 . I will make a release soon.

**Comment by omarkohl** [comment]
Released in 2.0.1
